### PR TITLE
Update hu.json

### PR DESCRIPTION
**Pull request by ovari**

### CODE REVIEW

File: i18n/hu.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
     "_.code": "hu",
-    "_.name": "Magyar (hu)",
+    "_.name": "magyar (hu)",
```
Comment (knadh) [inline i18n/hu.json:3]: I checked the Wikipedia entry and captializing `M` seems to be correct. Could you please explain this change?

Comment (ovari) [reply i18n/hu.json:3]: Thank you for your response.

In Hungarian, all languages are spelt with a lowercase letter.

The `suomi` languages also spells languages with a lowercase letter.

Below is a screenshot of the langues selector at the bottom of the [Thunderbird webpage](https://www.thunderbird.net/).
image

Thank you once again.

**Comment by knadh** [comment]
There are a few merge conflicts now owing to the big release yesterday. Please address them and we can merge. Thank you.

**Comment by ovari** [comment]
> There are a few merge conflicts now owing to the big release yesterday. Please address them and we can merge. Thank you.

How can this be resolved using the web browser to fix?

Can you please fix them if it is not too inconvenient for you?

Thank you

**Comment by ovari** [comment]
> There are a few merge conflicts now owing to the big release yesterday. Please address them and we can merge. Thank you.

If you can fix and merge and then if there are new strings to fix, can do in another patch.

Sorry, don't have an IDE. Only using the web browser to provide patches.

Thank you once again.

**Comment by ovari** [comment]
Should be done. Thank you

**Pull request merged**